staging/src/k8s.io/client-go/examples/fake-client/main_test.go

```diff
@@ -34,8 +36,24 @@ func TestFakeClient(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	watcherStarted := make(chan struct{})
 	// Create the fake client.
 	client := fake.NewSimpleClientset()
+	// A catch-all watch reactor that allows us to inject the watcherStarted channel.
+	client.PrependWatchReactor("*", func(action clienttesting.Action) (handled bool, ret watch.Interface, err error) {
+		gvr := action.GetResource()
+		ns := action.GetNamespace()
+		watch, err := client.Tracker().Watch(gvr, ns)
+		if err != nil {
+			return false, nil, err
+		}
+		// If you have informers watching multiple resources, you will need
+		// per-resource channels, or use a wait group if you don't know
+		// what resources are watched beforehand. See https://github.com/kubernetes/kubernetes/pull/95897
+		// for more examples.
```

Comment:
Let's either expand it to a full example or remove this comment.

IMHO, it's easy for the readers to come up with their own way to expand this example to cover the "multiple informers" case, I would just remove this comment.

Reply:
Done. Thanks for the suggestion. PTAL :)